src/actions/arcActions.ts

```diff
@@ -339,20 +332,18 @@ async function getProposalDetails(daoInstance: Arc.DAO, votingMachineInstance: A
     redemptions: [],
     reputationChange: Util.fromWei(contributionProposal.reputationChange),
     periodLength: Number(contributionProposal.periodLength),
-    preBoostedVotePeriodLimit,
+    preBoostedVotePeriodLimit: Number(votingMachineParams[1]),
```

Comment:
can get from `proposalDetails`

Reply:
no, this is a param on the voting machine, not on the proposal